curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3477ae9675db8bb7655b30.md

```diff
@@ -18,6 +18,13 @@ const hasH1 = new __helpers.CSSHelp(document).getStyle('h1');
 assert(hasH1);
 ```
 
+You should not add a new `style` tag. Work within the existing `style` tag.
```

Comment:
Perhaps this:

```suggestion
You should not add a new `style` tag. Add the new CSS rules to the existing `style` tag.
```

Reply:
I already commit the hint and the instructions suggested.